huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go

```diff
@@ -437,7 +440,34 @@ func resourceLoadBalancerV3Delete(ctx context.Context, d *schema.ResourceData, m
 		return diag.FromErr(err)
 	}
 
-	return nil
+	var diags diag.Diagnostics
+
+	// delete the EIP if necessary
+	eipID := d.Get("ipv4_eip_id").(string)
+	if _, ok := d.GetOk("iptype"); ok && eipID != "" {
+		eipClient, err := config.NetworkingV1Client(region)
+		if err == nil {
+			if eipErr := eips.Delete(eipClient, eipID).ExtractErr(); eipErr != nil {
+				if _, ok := err.(golangsdk.ErrDefault404); !ok {
+					eipDiag := diag.Diagnostic{
+						Severity: diag.Warning,
+						Summary:  "failed to delete EIP",
+						Detail:   fmt.Sprintf("failed to delete EIP %s: %s", eipID, eipErr),
+					}
+					diags = append(diags, eipDiag)
```

Comment:
without any other errors append to, why not just return

Reply:
we can seems this scenario be success with a warning message